src/main/java/racing/domain/Car.java

```diff
@@ -4,25 +4,26 @@
 
 public class Car {
     private static final int DEFAULT_POSITION = 1;
+
     private String name;
     private int position;
 
-    public Car() {
-        position = DEFAULT_POSITION;
+    public Car(Car car) {
+        this.name = car.getName();
+        this.position = car.getPosition();
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/java-racingcar/pull/649#discussion_r401321630

```suggestion
        this(car.name, car.position);
```

Reply:
앗 아직 체화가 안되었네요ㅠㅠㅠ
꼼꼼한 확인 감사합니다! 피드백 반영할게요 :)